process/block/metablock.go

```diff
@@ -945,29 +1240,7 @@ func (mp *metaProcessor) createShardInfo(
 		return nil, process.ErrAccountStateDirty
 	}
 
-	if !haveTime() {
-		log.Info(fmt.Sprintf("time is up after entered in createShardInfo method\n"))
-		return shardInfo, nil
-	}
-
-	mbHdrs := uint32(0)
-
-	timeBefore := time.Now()
-	orderedHdrs, orderedHdrHashes, sortedHdrPerShard, err := mp.getOrderedHdrs(round)
-	timeAfter := time.Now()
-
-	if !haveTime() {
-		log.Info(fmt.Sprintf("time is up after ordered %d hdrs in %v sec\n", len(orderedHdrs), timeAfter.Sub(timeBefore).Seconds()))
-		return shardInfo, nil
-	}
-
-	log.Debug(fmt.Sprintf("time elapsed to ordered %d hdrs: %v sec\n", len(orderedHdrs), timeAfter.Sub(timeBefore).Seconds()))
-
-	if err != nil {
-		return nil, err
-	}
-
-	log.Info(fmt.Sprintf("creating shard info has been started: have %d hdrs in pool\n", len(orderedHdrs)))
+	log.Info(fmt.Sprintf("creating shard info has been started \n"))
 
 	// save last committed header for verification
```

Comment:
You can delete lines between 1245-1254 and line 1237

Reply:
done